### ISSUE TITLE
Error relative referece safari-14-idb-fix

### ISSUE THREAD

**Issue by pittchuu** [jakearchibald/idb-keyval#123]
Hi,

I have this error when I import the module 👍 

Uncaught TypeError: Failed to resolve module specifier "safari-14-idb-fix". Relative references must start with either "/", "./", or "../".

**Comment by jakearchibald** [comment]
Can you give me more details on your setup?

**Comment by jakearchibald** [comment]
Ideally, a reduced reproduction of the issue

**Comment by soderlind** [comment]
I'm getting the same error.  I'm using Brave on macOS 11.5.1

I'm enqueueing my script i in WordPress using:

```php
add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\action_admin_enqueue_scripts' );
add_filter( 'script_loader_tag',__NAMESPACE__ . '\\filter_script_loader_tag', 10, 3 );

function action_admin_enqueue_scripts( string $hook_suffix ) : void {
	wp_register_script( 'super-admin-sites-menu', plugin_dir_url( __FILE__ ) . 'include/load-sites.js', [ 'admin-bar' ], '1.1.0', true );
	wp_enqueue_script( 'super-admin-sites-menu' );
}

// add type=module to script tag
function filter_script_loader_tag( $tag, $handle, $src ) {

	if ( 'super-admin-sites-menu' === $handle ) {
		return '<script type="module" src="' . esc_url( $src ) . '"></script>';
	}

	return $tag;
}
```

This is the start of `load-sites.js`:

```javascript

import {
	get,
	set,
	getMany,
	setMany,
	update,
	del,
	clear,
	keys,
	values,
	entries,
	createStore,
} from 'https://cdn.jsdelivr.net/npm/idb-keyval@5/dist/esm/index.js';


class LazyLoadSites {

	constructor() {
		this.sitesContainer = document.querySelector('.load-more')
		this.offsetStore = document.querySelector('#load-more-offset')
		this.observedContainer = document.querySelector("#wp-admin-bar-load-more")
		this.adminBar = document.getElementById('wpadminbar')
		this.observer = {}

	}

// more methods

}
```

**Comment by jakearchibald** [comment]
Ah, ok, it seems like jsdelivr doesn't handle dependencies properly. I've switched the recommendation to unpkg.

---

These built versions are also available on unpkg, e.g.:

```html
<script src="https://unpkg.com/idb-keyval@5/dist/iife/index-min.js"></script>
<!-- Or in modern browsers: -->
<script type="module">
  import { get, set } from 'https://unpkg.com/idb-keyval@5?module';
</script>
```

---

Fixed in 92bf4cbd75bdc12553fd8c4df94bf43b77dba81e
